<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::ThemeEditor::ThemeEditorWidget</class>
 <widget class="QWidget" name="Core::Internal::ThemeEditor::ThemeEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Theme Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Core::Internal::ThemeEditor::ThemeColorsTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Core::Internal::ThemeEditor::ThemeColorsTableView</class>
   <extends>QTableView</extends>
   <header location="global">coreplugin/themeeditor/themecolorstableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
